File: problems/problems_2092/solution.ts
function findAllPeople(n: number, meetings: number[][], firstPerson: number): number[] {
    
};

export function Solve(inputJsonElement: string): any {
	const inputValues: string[] = inputJsonElement.split("\n");
	const n: number = JSON.parse(inputValues[0]);
	const meetings: number[][] = JSON.parse(inputValues[1]);
	const firstPerson: number = JSON.parse(inputValues[2]);
	return findAllPeople(n, meetings, firstPerson);
}

File: problems/problems_2092/solution.go
package problem2092

import (
	"encoding/json"
	"log"
	"strings"
)

func findAllPeople(n int, meetings [][]int, firstPerson int) []int {
    
}

func Solve(inputJsonValues string) any {
	inputValues := strings.Split(inputJsonValues, "\n")
	var n int
	var meetings [][]int
	var firstPerson int

	if err := json.Unmarshal([]byte(inputValues[0]), &n); err != nil {
		log.Fatal(err)
	}
	if err := json.Unmarshal([]byte(inputValues[1]), &meetings); err != nil {
		log.Fatal(err)
	}
	if err := json.Unmarshal([]byte(inputValues[2]), &firstPerson); err != nil {
		log.Fatal(err)
	}

	return findAllPeople(n, meetings, firstPerson)
}
